tutorial/README.md

```diff
@@ -0,0 +1,281 @@
+
+# TokenScript for your token
+These instructions will help you create a TokenScript interface for a smart contract.
+User interface code and business logic is organised in XML declarations for a TokenScript engine to render in a mobile wallet. Eg [AlphaWallet](https://alphawallet.com/).
+
+
+## Creating your token and UI widgets
+
+* Open studio.ethereum.org, and start the Coin project
+  * It shows the smart contract and front-end files, and runs a vm and the Coin files in the browser
+
+Although not conforming to the ERC20 interface, a few changes to the example code will have it demonstrable in AlphaWallet
+* Add the following to the Coin contract
+```
+contract Coin {
+	//...
+
+    string public name = "Coin";
+    uint256 public decimals = 18;
+    string public symbol = "COIN";
+
+	// Returns balance of an address
+    function balanceOf(address addr) public view returns (uint) {
+        return balances[addr];
+    }
+
+```
+* lastly rename the `function send(` to `function transfer(`
```

Comment:
?

Reply:
The Ethereum Studio example uses `send` (presumably to be more approachable), but the function needs to be "transfer" to work with the default TokenScript "Send" action (that calls `transfer`). I hope this transfer of knowledge hasn't sent more confusion :p